Tests/Resources/app/config/config.php

```diff
@@ -5,3 +5,10 @@
 
 $loader->import(CMF_TEST_CONFIG_DIR.'/default.php');
 $loader->import(CMF_TEST_CONFIG_DIR.'/phpcr_odm.php');
+
+$config = array(
+    'form' => array(
+        'csrf_protection' => false,
+    ),
+);
+$container->loadFromExtension('framework', $config);
```

Comment:
that is ok then. i was just confused why that was not an issue until
now. but if your test is the first to actually submit a form then it
makes sense. and it does make sense to test actually submitting the form